source/common/common/dns_utils.cc

```diff
@@ -15,6 +15,7 @@ getDnsLookupFamilyFromCluster(const envoy::config::cluster::v3::Cluster& cluster
 Network::DnsLookupFamily
 getDnsLookupFamilyFromEnum(envoy::config::cluster::v3::Cluster::DnsLookupFamily family) {
   switch (family) {
+    PANIC_ON_PROTO_ENUM_SENTINEL_VALUES;
```

Comment:
For my understanding, when do we need this? Why don't we specify this for the enum in matchers.cc for example?

Reply:
More detail in linked issue, but proto creates sentinel values so it can test to see if you go beyond MIN and MAX, and if you want to do an exhaustive switch you need to handle those.

If you don't do exhaustive switch you end up introducing bugs when you add new fields, like the fact this switch was missing handling for ALL and would have panic'd if that config path was hit in production :-(